Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Drop winsqlite3 feature

We are not shipping a winsqlite3 feature with the libsqlite3-sys crate
in debian, so also remove it here.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -197,7 +197,6 @@
 vtab = ["libsqlite3-sys/min_sqlite_version_3_7_7"]
 wasm32-wasi-vfs = ["libsqlite3-sys/wasm32-wasi-vfs"]
 window = ["functions"]
-winsqlite3 = ["libsqlite3-sys/winsqlite3"]
 with-asan = ["libsqlite3-sys/with-asan"]
 
 [badges.appveyor]
